src/main/java/science/icebreaker/network/NetworkController.java

```diff
@@ -0,0 +1,61 @@
+package science.icebreaker.network;
+
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.Arrays;
+import java.util.List;
+
+
+@RestController
+@RequestMapping("/network")
+public class NetworkController {
+
+    private final NetworkService service;
+
+
+    public NetworkController(NetworkService service) {
+        this.service = service;
+    }
+
+
+    @GetMapping("")
+    @ApiOperation("Returns the number of nodes - just for fun. This endpoint can change any time and should not be " +
+            "actually used.")
+    public int getNumberOfNodes() {
+        return service.countNodes();
+    }
+
+
+    @GetMapping("/node")
```

Comment:
If it returns more than one, I would name the path "nodes".

Reply:
The names of the other endpoints such as `/wiki` and `/device_availability` are also in the singular form even though they return multiple values, so I would like to keep it the same way.